templates/page/campus/home.html.twig

```diff
@@ -63,9 +76,9 @@ gratuite et ouverte à tous.' %}
                 <div class="campus__text">
                     <h2>Se former</h2>
                     <p>
-                        Vous trouverez sur cette page nos cours sur la vie politique. 
-                        Ces « microlearning » sont des parcours éducatifs et ludiques à suivre en ligne, 
-                        à votre rythme, pour mieux comprendre le fonctionnement de la vie politique 
+                        Vous trouverez sur cette page nos cours sur la vie politique.
+                        Ces « microlearning » sont des parcours éducatifs et ludiques à suivre en ligne,
```

Comment:
You wrote "microlearnings" with a "s" here : https://github.com/EnMarche/en-marche.fr/pull/2553/files#diff-1a752ab87d882a61d58251f2e3f00e07R75

but not here?